### ISSUE TITLE
Weekly top reviewers points miscalculated

### ISSUE THREAD

**Issue by rctgamer3** [mozilla/addons-server#6949]
I was wondering why the #1 reviewer for this week has such a large number of points listed.
Moderated the user review queue this afternoon and noted points received were multiplied * 3 there - so for example one review would make the number jump from 100 to 103 instead of 101 - but actual points received was still the actual amount, one.

Possibly related to #6573 - that had to do with what permissions are set for individual reviewers. Leaderboard scores are now also multiplied * 3 (instead of 2) - so if our accounts indeed have an unspecified 3 permissions set - that would confirm my hunch.

**Comment by wagnerand** [comment]
It turns out that the leaderboard calculations are wrong (which is used in different places). @ValentinaPC has done some testing as well, thank you for that!

**Comment by wagnerand** [comment]
@ValentinaPC should have the details for testing this.

tl;dr: Have a reviewer in multiple reviewer groups at once and check scores in leaderboards when doing post-reviews.

Let me know if you need more detailed steps to test.

**Comment by ValentinaPC** [comment]
Verified calculation of reviewer recognition points using AMO-dev FF57(Win 7).

Added a user in 2 groups -  Reviewers: Content and Reviewers: Add-ons
Reviewed (confirm approval, reject multiple versions, approve cotent) add-ons with all possible weights and points were generated as expected.
Also, sections - Recent Points/Top Reviewers (past 7 days)/All Time - and leaderboars display correct amount of points.

We can mark this as verified-fixed unless @rctgamer3 needs other details or wants to check it himself.
